ndex2/client.py

```diff
@@ -793,8 +1080,27 @@ def set_provenance(self, network_id, provenance):
 
     def set_read_only(self, network_id, value):
         """
+
         Sets the read only flag on the specified network
 
+        .. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		network_id = 'INSERT NETWORK UUID'
+
+		#set network to read only
+		my_ndex.set_read_only(network_id, True)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client